src/service/cli/fill.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+const chalk = require(`chalk`);
+const fs = require(`fs`).promises;
+const { getRandomInt, shuffle } = require(`../utils`);
+
+const DEFAULT_COUNT = 1;
+const MAX_PUBLICATION_COUNT = 1000;
+const MAX_ANNOUNCE_SENTENCES_COUNT = 5;
+const MIN_COMMENTS_COUNT = 2;
+const MAX_COMMENTS_COUNT = 4;
+const MAX_ANNOUCE_LENGTH = 250;
+const MAX_TEXT_LENGTH = 1000;
+const FILE_NAME = `fill-db.sql`;
+
+const FILE_SENTENCES_PATH = `./data/sentences.txt`;
+const FILE_TITLES_PATH = `./data/titles.txt`;
+const FILE_CATEGORIES_PATH = `./data/categories.txt`;
+const FILE_COMMENTS_PATH = `./data/comments.txt`;
+
+const users = [
+    {
+      email: `ivanov@example.com`,
+      passwordHash: `5f4dcc3b5aa765d61d8327deb882cf99`,
+      firstName: `Иван`,
+      lastName: `Иванов`,
+      avatar: `avatar1.jpg`
+    }, {
+      email: `petrov@example.com`,
+      passwordHash: `5f4dcc3b5aa765d61d8327deb882cf99`,
+      firstName: `Пётр`,
+      lastName: `Петров`,
+      avatar: `avatar2.jpg`
+    }
+];
+
+const PictureRestrict = {
+    MIN: 1,
+    MAX: 16,
+};
+
+const getPictureFileName = (index) => (`item${index.toString().padStart(2, '0')}.jpg`);
+
+const generateComments = (count, publicationId, userCount, comments) => (    
+    Array(count).fill({}).map(() => ({
```

Comment:
```suggestion
    new Array(count).fill({}).map(() => ({
```
конструкторы должны вызываться всегда с `new`